### PR TITLE
Changes to make the compile with swift3

### PR DESCRIPTION
**Pull request by jamiltz**
@pasin with those changes the app runs but there are crashes. Some of them were because the target actions from storyboards to files were removed (probably by the Swift3 code migrator).

Apart from those there's a crash when adding a list https://gist.github.com/jamiltz/feae0a6d4cfa661a1d5e1d030d58de87

Any idea what might be causing this?

### CODE REVIEW

File: Todo/TaskListsViewController.swift [hunk 1]
```diff
@@ -92,7 +92,7 @@ class TaskListsViewController: UITableViewController, UISearchResultsUpdating {
         
         let row = listRows![indexPath.row] as CBLQueryRow
         
-        cell.textLabel?.text = row.key as? String
+        //cell.textLabel?.text = row.key as? String
```
Comment (pasin) [inline Todo/TaskListsViewController.swift]: Should we uncommented this?

File: Todo/TaskListsViewController.swift [hunk 2]
```diff
@@ -218,12 +218,14 @@ class TaskListsViewController: UITableViewController, UISearchResultsUpdating {
...
         let rows = incompTasksCountsLiveQuery.rows
         while let row = rows?.nextRow() {
-            if let listId = row.key as? String, let count = row.value as? Int {
-                counts[listId] = count
-            }
+            print(row)
+            print(counts)
+            //if let listId = row.key as? String, let count = row.value as? Int {
+            //    counts[listId] = count
+            //}
```
Comment (pasin) [inline Todo/TaskListsViewController.swift]: Are these commented out lines and prints for debugging?

Comment (jamiltz) [inline Todo/TaskListsViewController.swift]: Yes, but when line 226 - 228 are uncommented it doesn't build on my laptop.

File: Todo/TaskListsViewController.swift (hunk 1 above)
Comment (pasin) [inline Todo/TaskListsViewController.swift]: row.key is conflicted with row.key(atIndex) method. I think a temp workaround would be 'row.value(forKey: "key")'. Can you try that?

File: Todo/TaskListsViewController.swift (hunk 2 above)
Comment (pasin) [inline Todo/TaskListsViewController.swift]: I think a temp workaround would be 'row.value(forKey: "key")'. Can you try that?

**Pull request merged**